src/cats_queue/consumer.py

```diff
@@ -46,23 +48,27 @@ def set_stop_processing(self, *args, **kwargs):
     def execute(self, msg: ConsumerRecord):
         topic = msg.topic
         message = msg.value
+
         if topic == self.FIND_CAT_TOPIC:
-            self.inference.add_user(Person(_id=None, path=message['image_path'], quadkey=message["quadkey"],
+            self.inference.add_user(Person(_id=None, paths=message['image_paths'], quadkey=message["quadkey"],
                    embeddings=None, chat_id=message["user_id"]))
         elif topic == self.SAW_CAT_TOPIC:
-            self.inference.save_new_cat(Cat(_id=None, path=message['image_path'], quadkey=message["quadkey"],
+            self.inference.save_new_cat(Cat(_id=None, paths=message['image_paths'], quadkey=message["quadkey"],
                                            embeddings=None, additional_info=message["additional_info"]))
 
     def main_loop(self):
         while not self.stop_processing:
             for msg in self.consumer:
+
```

Comment:
could you please add linting to project
like https://py-vscode.readthedocs.io/en/latest/files/linting.html
for example black

it's not ok when everyone try to refactor as they want

Reply:
ok

Reply:
have you added?